src/index.js

```diff
@@ -1,28 +1,43 @@
 const express = require("express");
 const mongoose = require("mongoose");
 const env = require("dotenv");
-const ngoRoutes = require("../routes/ngo");
 const cors =require('cors')
 const app = express();
+const session = require("express-session");
+const passport = require("passport");
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+const ngoRoutes = require("../routes/ngo");
+const userRoutes = require("../routes/user");
+const Ngo = require("../models/ngo");
+const User = require("../models/user");
 
 env.config({path: "../.env"});
 
-const db_user = process.env.db_user;
-const db_pwd = process.env.db_pwd;
-const db_id = process.env.db_id;
-const db_name = process.env.db_name;
+const dbUser = process.env.dbUser;
+const dbPwd = process.env.dbPwd;
+const dbId = process.env.dbId;
+const dbName = process.env.dbName;
```

Comment:
environment constants should be all caps.
Because that's how the shell is supposed to be seeing them.